config/application.rb

```diff
@@ -62,6 +62,7 @@ class Application < Rails::Application
 
     config.action_controller.perform_caching = true
     config.cache_store = :memory_store
```

Comment:
We're still using the memory_store. Given we're caching all in-progress applications, we should probably switch to using Redis and merge this PR after that switch has happened. 

There's [some guidance](https://github.com/mperham/sidekiq/wiki/Using-Redis#multiple-redis-instances) out there recommending the use of separate Redis instances for Sidekiq and the cache store (with persistence and an LRU eviction policy configured, respectively). Is that something we can start getting set up on the infrastructure side of things, or does this need more discussion about options?

Reply:
we can create a separate redis instance solely for caching purpose with a different eviction policy.